person.rb

```diff
@@ -1,4 +1,6 @@
-class Person
+require './nameable'
```

Comment:
- [ ] @tarikbouari kindly use `require_relative` as its the ruby best practice, require is used when calling external files like gems